lib/rubber/recipes/rubber/setup.rb

```diff
@@ -90,7 +91,9 @@ def enable_root_ssh(ip, initial_ssh_user)
     end
 
     task :_allow_root_ssh, :hosts => "#{initial_ssh_user}@#{ip}" do
-      rsudo "mkdir -p /root/.ssh && cp /home/#{initial_ssh_user}/.ssh/authorized_keys /root/.ssh/"
+      if initial_ssh_user != 'root'
+        rsudo "mkdir -p /root/.ssh && cp ~/.ssh/authorized_keys /root/.ssh/"
```

Comment:
I think the use of `~` is probably for non-standard home directories. Not sure how standard `$HOME` is, but `$HOME` has been more reliable in my experience.